inst/templates/cran/msrv.R

```diff
@@ -0,0 +1,116 @@
+# read the DESCRIPTION file
+desc <- read.dcf("DESCRIPTION")
+
+if (!"SystemRequirements" %in% colnames(desc)) {
+  fmt <- c(
+    "`SystemRequirements` not found in `DESCRIPTION`.",
+    "Please specify `SystemRequirements: Cargo (Rust's package manager), rustc`"
+  )
+  stop(paste(fmt, collapse = "\n"))
+}
+
+# extract system requirements
+sysreqs <- desc[, "SystemRequirements"]
+
+# check that cargo and rustc is found
+if (!grepl("cargo", sysreqs, ignore.case = TRUE)) {
+  stop("You must specify `Cargo (Rust's package manager)` in your `SystemRequirements`")
+}
+
+if (!grepl("rustc", sysreqs, ignore.case = TRUE)) {
+  stop("You must specify `Cargo (Rust's package manager), rustc` in your `SystemRequirements`")
+}
+
+# split into parts
+parts <- strsplit(sysreqs, ", ")[[1]]
+
+# identify which is the rustc
+rustc_ver <- parts[grepl("rustc", parts)]
+
+# perform checks for the presence of rustc and cargo on the OS
+no_cargo_msg <- c(
+  "----------------------- [CARGO NOT FOUND]--------------------------",
+  "The 'cargo' command was not found on the PATH. Please install Cargo",
+  "from: https://www.rust-lang.org/tools/install",
+  "",
+  "Alternatively, you may install Cargo from your OS package manager:",
+  " - Debian/Ubuntu: apt-get install cargo",
+  " - Fedora/CentOS: dnf install cargo",
+  " - macOS: brew install rustc",
+  "-------------------------------------------------------------------"
```

Comment:
Out of curiosity, is this message template standard? If not, should we include windows here as well?

Reply:
I don't know the origin of it. I am thinking we should actually modify this message and the one below to recommend using rustup and not cargo or rustc separately. Do you agree?